CMakeLists.txt

```diff
@@ -84,13 +84,13 @@ set(bullet_GIT_REPOSITORY https://github.com/RobotLocomotion/bullet-pod.git)
 set(bullet_GIT_TAG 4319ffd7e9251066d93064f5a8dab12e33dbe5e2)
 set(bullet_IS_PUBLIC TRUE)
 set(bullet_IS_CMAKE_POD TRUE)
-set(cmake_GIT_REPOSITORY https://github.com/RobotLocomotion/cmake.git)
-set(cmake_GIT_TAG 3ca531594097919f0c9784585ecb32fe0df15404)
+set(cmake_GIT_REPOSITORY https://github.com/rdeits/cmake.git)
+set(cmake_GIT_TAG 25149772a2734589d3a925bc62f92ba07e6f3a97)
```

Comment:
I think that probably the right procedure is:

1. open pull request in the external
1. open pull request in drake pointing to fork
1. when drake pull request passes review & tests, then merge the external pull request and update drake to point to master on the external
1. wait for one more round of unit tests on drake